persistence-dse-6.8/src/main/java/io/stargate/db/dse/impl/interceptors/ProxyProtocolQueryInterceptor.java

```diff
@@ -187,45 +190,50 @@ public void register(EventListener listener) {
     wrapped.ifPresent(w -> w.register(listener));
   }
 
-  private void resolvePeers() {
+  @SuppressWarnings("NonAtomicOperationOnVolatileField")
+  private void resolveAllPeers() {
     if (!Strings.isNullOrEmpty(proxyDnsName)) {
       try {
-        Set<InetAddress> resolved = resolver.resolve(proxyDnsName);
+        publicPeers = resolvePeers(proxyDnsName, publicPeers);
+        privatePeers = resolvePeers(internalProxyDnsName, privatePeers);
+      } catch (UnknownHostException e) {
+        throw new ServerError("Unable to resolve DNS for proxy protocol peers table", e);
+      }
+      scheduler.schedule(this::resolveAllPeers, resolveDelaySecs, TimeUnit.SECONDS);
```

Comment:
From the code, I see that a typical value for `resolveDelaySecs` is `10 seconds`, is that true? 
Is it good enough in the Stargate context?

Reply:
Yes it's 10 seconds. I didn't pick that value, it's what was already there.

@mpenick any insight on how it was chosen?

Reply:
I'm sorry I don't have as much insight as I should for the value of the delay. I believe that was the existing value.